test/unittests/mongoBackend/mongoContextProvidersQuery_test.cpp

```diff
@@ -717,9 +800,12 @@ TEST(mongoContextProvidersQueryRequest, noPatternAttrsSubset)
 *
 * noPatternSeveralCREs -
 *
-* Discover:  E1 - no attrs
-* Result:    E1 - (A1, A2, A3) - http://cr1.com
-*            E1 - (A1, A4)     - http://cr2.com
+* Query:  E1 - no attrs
+* Result: E1 - A1, A2, A3 - http://cr1.com
+*              A4         - http://cr2.com
+*
+* For A1 two CPrs overlap and mongoBacken takes http://cr1.com
```

Comment:
Similar problem here, not so much for queries but definitely for updates.
Is an issue created to explain and remember these problems (this and the previous)?